input/resources/CodeSystem-ehealth-media-usage-mode.json

```diff
@@ -0,0 +1,54 @@
+{
+  "resourceType": "CodeSystem",
+  "id": "ehealth-media-usage-mode",
+  "url": "http://ehealth.sundhed.dk/cs/media-usage-mode",
+  "version": "0.6.0",
```

Comment:
Is there any reason for this specific version number? It looks like it is the same one used on all new CodeSystem/ConceptMap/ValueSet